packages/@adobe/spectrum-css-temp/components/barloader/index.css

```diff
@@ -18,15 +18,17 @@ governing permissions and limitations under the License.
   flex-flow: row wrap;
   justify-content: space-between;
   align-items: center;
-  width: var(--spectrum-barloader-large-width);
+  min-width: var(--spectrum-barloader-large-width);
```

Comment:
Just wanna double check: did we check w/ spectrum regarding this `min-width` here? I noticed that spectrum css has a min-width of 192px applied to their progress bars but only if it has a side label not if the label is on top. You can try it out by modifying the widths [here](https://opensource.adobe.com/spectrum-css/progressbar.html#customwidth)

IMO we should def have a min-width so that weird label and number value stacking doesn't happen but do we want a smaller value? 192px feels kinda large. This is 100px:
![image](https://user-images.githubusercontent.com/9661127/107447411-4f9aaa00-6af5-11eb-968b-f65ecdb2d4ab.png)